drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -1580,6 +1580,14 @@ static int vc5_hdmi_init_resources(struct vc4_hdmi *vc4_hdmi)
 	if (IS_ERR(vc4_hdmi->dvp_regs))
 		return PTR_ERR(vc4_hdmi->dvp_regs);
 
+	res = platform_get_resource_byname(pdev, IORESOURCE_MEM, "intr2");
+	if (!res)
+		return -ENODEV;
+
+	vc4_hdmi->intr2_regs = devm_ioremap(dev, res->start, resource_size(res));
```

Comment:
dev_ioremap_resource? One ought to be able to combine the resource lookup and the ioremap, but I can't find such a convenience function.

Reply:
I don't believe there is, and it matches all the other mappings from Maxime's code.

Reply:
You can use devm_ioremap_resource here:
```
	vc4_hdmi->intr2_regs = devm_ioremap(dev, res);
```
I think it was the combined find and remap that @6by9 was agreeing doesn't exists, but for consistency I'll leave this as is.